src/App.svelte

```diff
@@ -0,0 +1,72 @@
+<script>
+  import { db } from "./firebase.js";
+  export let name;
+  let arrList = [];
+  let newItemText = "";
+  let newItemQuantity;
+
+  db.collection("grocery").onSnapshot((snapData) => {
+    arrList = snapData.docs;
+  });
+
+  function addItem() {
+    db.collection("grocery").add({
+      name: newItemText,
+      quantity: newItemQuantity,
+    }); // It adds new data to the collection grocery
+    newItemText = "";
+    newItemQuantity = "";
+  }
+
+  function removeItem(itemId) {
+    console.log("The event works");
+    db.collection("grocery").doc(itemId).delete();
+  }
+</script>
+
+<main>
+  <h1>Grocery list</h1>
+  <h2>
```

Comment:
This is invalid HTML, H2 does not support form controls (eg input) inside.

Reply:
What does "H2 does not support form controls (eg input) inside." mean?

Reply:
It means you cannot have elements like input inside h2, as you did. It needs to be separated elements. ex:

```
<h2>this is a title</h2>
<input type="text" />
```

Reply:
Oh Ok that's good to know.